client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx

```diff
@@ -261,55 +270,6 @@ const InnerContent = ({
 	);
 };
 
-interface CancellationCTAProps {
```

Comment:
if we're going to migrate all products to this v2 page maybe we should keep this - for products that don't have self service cancellation

Reply:
We can do although it will remain unused until then...unless I've missed some logic somewhere

Reply:
just thought that if we left it out when the rest of the products get migrated it could easily be forgotten

Reply:
happy to do this in later work 👍